RELEASE_NOTES.md

```diff
@@ -17,6 +17,19 @@ v1.0.0-alpha.xx
 
 ### Improvements
 
+- Added paged implementations of `getWithRelationship` and
```

Comment:
[consider][minor] Super minor, but this seems a bit verbose for release notes, and covers some stuff I'd expect to see in API docs rather than release notes.

But like I say, super minor subjective opinion.